extensions/amp-playbuzz/0.1/amp-playbuzz.js

```diff
@@ -181,8 +184,9 @@ class AmpPlaybuzz extends AMP.BaseElement {
       this.iframeLoaded_ = true;
       this.attemptChangeHeight(this.itemHeight_).catch(() => {/* die */ });
 
-      const unlisten = this.getViewport().onChanged(
+      const unlisten = events.listen(this.win,'scroll',
```

Comment:
`scroll` event on window won't give you scroll events in most of cases. You can experiment with `viewport.onScroll`. Though I'd first like to understand why `onChanged` is not fast enough for you - it's used for some pretty critical infra work that also requires certain fidelity.